config/templates/cluster.yaml

```diff
@@ -68,6 +68,10 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 # Number of I/O operations per second (IOPS) that the controller node disk supports. Leave blank if controllerRootVolumeType is not io1
 #controllerRootVolumeIOPS: 0
 
+# Tenancy of the controller nodes. Options are "default" and "dedicated"
+# Documentation: http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/dedicated-instance.html
```

Comment:
I appreciate your documentation here. Linking to the original source of information is always good idea 👍